js/templates/search.es6.js

```diff
@@ -1,14 +1,11 @@
 const bel = require('bel');
 
 module.exports = function () {
-    return bel`<div class="js-search js-menu-section">
-        <form class="js-search-form" name="x">
-          <input type="text" autocomplete="off" placeholder="Search DuckDuckGo" name="q" class="js-search-input" value="${this.model.searchText}" />
```

Comment:
yeah, sorry for the confusion. this is correct. I think we'll add our ajax autocomplete later, so we should keep `autocomplete="off"` in this template. Agree with your first comment about tracking.